.golangci.yaml

```diff
@@ -26,11 +28,20 @@ linters:
     - wastedassign
     - whitespace
     - protogetter
-    # - gocritic
+    # TODO enable https://github.com/openfga/openfga/issues/598
+    # - bodyclose
+    # - contextcheck
     # - exhaustive
+    # - errorlint
+    # - gochecknoglobals
+    # - gocritic
+    # - godot
+    # - gosec
     # - noctx
-    # - promlinter
-
+    # - inamedparam
+    # - perfsprint
+    # - prealloc
+    # - tparallel
```

Comment:
i went thru the list in https://golangci-lint.run/usage/linters/#disabled-by-default

the ones commented out satisfy both:
1. they seem useful
2. they report issues in our repo